prow/tide/status_test.go

```diff
@@ -393,8 +451,19 @@ func TestSetStatuses(t *testing.T) {
 			t.Fatalf("Failed to get log output before testing: %v", err)
 		}
 
-		sc := &statusController{ghc: fc, gc: &git.Client{}, config: ca.Config, logger: log}
-		sc.setStatuses([]PullRequest{pr}, pool, blockers.Blockers{})
+		sc, err := newStatusController(
+			log,
```

Comment:
I don't really see the need for this to be on 10 lines or whatever, maybe put all the args on the same line? (and elsewhere)